stdlib/moreLabels.mli

```diff
@@ -955,6 +955,11 @@ module Map : sig
       val is_empty: 'a t -> bool
       (** Test whether a map is empty or not. *)
 
+      val is_singleton: 'a t -> bool
+      (** Test whether a map has exactly one element or not.
+
+        @since 5.5 *)
+
```

Comment:
You need to modify the files in `stdlib/templates` instead (read the comment at the top of the file).